packages/cli/src/utils/memory.js

```diff
@@ -0,0 +1,13 @@
+import chalk from 'chalk'
+import consola from 'consola'
+import prettyBytes from 'pretty-bytes'
+
+export function getMemoryUsage() {
+  // https://nodejs.org/api/process.html#process_process_memoryusage
+  const { heapUsed, rss } = process.memoryUsage()
+  return `Memory usage: ${chalk.bold(prettyBytes(heapUsed))} (RSS: ${prettyBytes(rss)})`
+}
+
+export function showMemoryUsage() {
+  consola.info(getMemoryUsage())
```

Comment:
And here I will add the text :)

```suggestion
  const { heap, rss } = getMemoryUsage()
  consola.info(`Memory usage: ${chalk.bold(prettyBytes(heap))} (RSS: ${prettyBytes(rss)})`)
```